sale_margin_percentage/models/res_company.py

```diff
@@ -4,4 +4,4 @@
 class ResCompany(models.Model):
     _inherit = "res.company"
 
-    margin_threshold = fields.Float(digits=(16, 2), help="Margin Threshold for product in order lines")
+    margin_threshold = fields.Float(digits=(18, 4), help="Margin Threshold for product in order lines")
```

Comment:
Why this one?

Reply:
I thought it was necessary because instead of being a percentage, 50% for example, now it must be used as a ration, 0.5 in the same example. But checking [the native field](https://github.com/odoo/odoo/blob/15.0/addons/sale_margin/models/sale_order.py#L12) I think is better to remove it.